recipes/domclust/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: domclust 
+  version: "1.2.7l"
+
+about:
+  home: http://mbgd.genome.ad.jp/domclust/
+  license: 
+  summary: Effective tool for orthologous grouping in multiple genomes 
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  fn: domclust.tgz 
+  url: "http://mbgd.genome.ad.jp/domclust/domclust.tgz"
+  sha256: 9e938027f2a0b859feaea5a7701c9b7cd41d494a49cb7bd66890077dec551e3e 
+
+requirements:
+  build:
+    # linux build
+    - gcc # [linux]
+    - perl
+    # osx build
+    #- llvmdev ==3.9.1 # [osx]
+    #- libcxx # [osx]
+    #- boost ==1.63.0 # [osx]
+    # general requirements
+  run:
+    # linux build
+    - libgcc # [linux]
+    #- libcxx # [osx]
+    - perl
+    #- boost ==1.63.0 # [osx]
+
+test:
+  commands:
+    - domclust -h
+    - domclust $PREFIX/share/test.hom $PREFIX/share/test.gene
```

Comment:
PREFIX does not exists in the test data section (in Docker). You could:
- skip this test
- use a hack like `which domclust`/../share/test.hom
- run the test as part of the build.sh script